hw/ip/otbn/dv/smoke/smoke_test.s

```diff
@@ -119,34 +119,6 @@ csrrw x0, mod1, x23
 li x23, 0x53769ada
 csrrw x0, mod0, x23
 
```

Comment:
Why this change to the smoke test? Please don't explain it in code comments here since they would not make sense (they refer to something that is no longer there). Perhaps the smoke test is explained somewhere, and that can give some indication?